rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LRAStatus.java

```diff
@@ -25,6 +25,7 @@
 import org.eclipse.microprofile.lra.client.LRAInfo;
 
 import java.io.IOException;
+import java.util.Objects;
 
 //@Data
```

Comment:
should not be removed these commented annotations out?

Reply:
Same comment as before, so I will just raise a separate JIRA.

Reply:
agree, here this my comment is was rather "a meanwhile thing" as there is no change which links this PR

Reply:
I don't think we should remove things in this PR that are not related to the PR